source/extensions.html.erb

```diff
@@ -3,54 +3,67 @@ title: Extensions
+  <div class="container-fluid extensions-filter__navbar d-flex justify-content-center">
 
-    <div class="form-check form-check-custom">
-      <input type="checkbox" class="form-check-input" id="filter-community" name="community" checked>
-      <label class="form-check-label" for="filter-community">
-        <%= inline_svg("icons/checkbox.svg", class: "isvg") %>
-        Community
-      </label>
-    </div>
+  <input type="checkbox" class="form-check-input show-all__mobile show-all__mobile__input" id="show-all" name="Show all">
+  <label class="show-all__mobile show-all__mobile__label" for="show-all"></label>
 
-    <div class="form-check form-check-custom">
-      <input type="checkbox" class="form-check-input" id="filter-third-party" name="third-party" checked>
-      <label class="form-check-label" for="filter-third-party">
-        <%= inline_svg("icons/checkbox.svg", class: "isvg") %>
-        Third-Party
-      </label>
-    </div>
+    <ul class="extensions-filter__navbar__nav nav align-items-center">
+      <% extensions = Solidus::Extension.from_yaml_data(data.extensions, environment: environment) %>
+
+      <% group_extensions(extensions).each_pair.with_index do |(group_name, group_extensions), i| %>
+        <li class="nav-item" data-filter="<%= group_name.underscore.gsub('_', '-') %>">
+          <div class="form-check-custom pl-0">
+            <input type="checkbox" class="form-check-input extensions-filter__input" id="filter-<%= group_name.underscore.gsub(' ', '-') %>" name="<%= group_name.underscore.gsub(' ', '-') %>">
+            <label class="btn btn-sm extensions-filter__label" for="filter-<%= group_name.underscore.gsub(' ', '-') %>">
+              <%= group_name %>
+            </label>
+          </div>
+        </li>
+      <% end %>
+      <button class="js-more btn btn-sm extensions-filter__more">More</button>
+    </ul>
   </div>
 </div>
 
-<% extensions = Solidus::Extension.from_yaml_data(data.extensions, environment: environment) %>
-
 <div class="extensions-list-block content-block content-block-gray-100">
   <div class="container">
     <div class="alert alert-warning mb-0 d-none" role="alert">
       No extensions matching your criteria
     </div>
 
     <div class="accordion" id="accordion">
+      <div class="extensions-group accordion-item most-popular" id="extensions-group-most-popular">
+        <h3 class="title">
+          <a class="collapsed" href="#" data-toggle="collapse" data-target="#collapse-most-popular">
+            Most popular
+            <%= inline_svg("icons/arrow-down.svg", class: "isvg isvg-arrow") %>
+          </a>
+        </h3>
+
+        <div id="collapse-most-pupular" class="content collapse" data-parent="#accordion">
+          <div class="extensions">
+            <% data.extensions.each do |extensions, extension| %>
+                <% if extension[:most_popular] %>
+                  
+                <% end %>
+            <% end %>
+          </div>
```

Comment:
Hi @kennyadsl 👋 I need your help! I want to display the extensions which have `most_popular: true`. I've done it with the code above this comment but I cannot include the same partial we're using `_extension.html.erb`. I've found some rails conflicts about the `<% extension.categories.each do |category| %>` and `<% extension.ci.build_statuses(versions: versions).each_pair do |version, build| %>`. Maybe is there a better way to group extensions that have the `most_popular` value true? Should I create a new helper method like the following?
```
def group_popular(extensions)
    extensions.group_by(&:most_popular).sort_by { |group_name, _| group_name }.to_h
  end
```
(I've tried also this way, but there's always some errors like the ones I mentioned in this comment 😭 )

Reply:
The big picture is: we're going to show the `Most Popular` group/category as the first of the list, only when a user doesn't filter the extensions. So the single extension's card shown in this category will be repeated even into their own group (for example if I set as Most Popular the PayPal extension, that will be shown into the Most Popular group and even into the Payments group).